### ISSUE TITLE
Unexpected Error in the update Method of Feature Instance in FeatureStore

### ISSUE THREAD

**Issue by masayukeeeee** [googleapis/python-aiplatform#2287]
Executing the following code raises an AttributeError: Unknown field for Feature: operation. However, it seems like the update process itself is completed (the description and labels appear updated when checked on the GUI). This error message suggests that there's no attribute called 'operation' in the Feature object. Upon inspecting the code, could it be that the object returned by api_client.update() doesn't originally have the 'operation' attribute?

#### Steps to reproduce

  1. create a feature in a entity of FeatureStore
  2. try update method of the feature.

#### Code example

```python
fs = FeatureStore(featurestore_name="name")
entity = fs.get_entity_type(entity_type_id="entity_id")
feature = entity.get_feature("feature_id")
feature.update(
    labels = {"kind": "registered-attribution"}
)
```

#### Stack trace
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/tmp/ipykernel_1/3219449219.py in <module>
      2 feature.operation = {'name': 'hogehoge'}
      3 feature.update(
----> 4     labels = {"kind": "registered-attribution"}
      5 )

~/.local/lib/python3.7/site-packages/google/cloud/aiplatform/featurestore/feature.py in update(self, description, labels, request_metadata, update_request_timeout)
    247 
    248         _LOGGER.log_action_started_against_resource_with_lro(
--> 249             "Update", "feature", self.__class__, update_feature_lro
    250         )
    251 

~/.local/lib/python3.7/site-packages/google/cloud/aiplatform/base.py in log_action_started_against_resource_with_lro(self, action, noun, cls, lro)
    173         """
    174         self._logger.info(
--> 175             f"{action} {cls.__name__} {noun} backing LRO: {lro.operation.name}"
    176         )
    177 

/opt/conda/lib/python3.7/site-packages/proto/message.py in __getattr__(self, key)
    746         if pb_type is None:
    747             raise AttributeError(
--> 748                 "Unknown field for {}: {}".format(self.__class__.__name__, key)
    749             )
    750         pb_value = getattr(self._pb, key)

AttributeError: Unknown field for Feature: operation
```

**Comment by matheusMoreno** [comment]
+1. Got this error today. The `update()` method calls a method from the low-level API client called `update_feature()`, which returns a `Feature` object. However, the method expects it to be an `Operation` object with an `operation` attribute.

**Comment by matthew29tang** [comment]
Thanks for reporting this, I'm working on a fix for this in #2520.

**Comment by matthew29tang** [comment]
#2520 was merged and will be included in the next release.
